src/test/resources/application.properties

```diff
@@ -65,6 +65,10 @@ results.encounterType.uuid=7aa974b5-7523-11eb-8077-0242ac120009
 
 # Lab Order Type (Bahmni specific)
 bahmni.test.orderType.uuid=7a82584d-7523-11eb-8077-0242ac120009
+
+# Base fhir2 relative uri, it should default to 'ms/fhir2Servlet' because of https://issues.openmrs.org/browse/FM2-447
+# When FM2-447 has been resolved, this can change to the standard relative uri 'ws/fhir2/R4'
+base.relative.fhir2.api.uri=ms/fhir2Servlet
```

Comment:
What would be better is to have a `{{fhirR4.baseUrl}}` alias that takes us to the right place, so basically that would be the concatenation already of `{{openmrs.baseUrl}}` and `/ms/fhir2Servlet`.

Also keep a commented out line that points to what this should become in the future.